integration-tests/validation/src/main/java/com/android/designcompose/testapp/validation/examples/AllExamples.kt

```diff
@@ -29,6 +29,7 @@ val EXAMPLES: ArrayList<Triple<String, @Composable () -> Unit, String?>> =
         Triple("HelloVersion", { HelloVersion() }, HelloVersionDoc.javaClass.name),
         // Alphabetically ordered and trying to put similar tests together...
         Triple("Alignment", { AlignmentTest() }, AlignmentTestDoc.javaClass.name),
+        Triple("AutoLayout", { AutoLayoutTest() }, AutoLayoutTestDoc.javaClass.name),
```

Comment:
We have a few tests that test various aspects of autolayout. Is there a reason this one is called AutoLayout? If not can we rename it to MinMax or something?